api/v1/model/src/commonMain/kotlin/FilterOperatorAndValue.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2024 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.eclipse.apoapsis.ortserver.api.v1.model
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+/**
+ * Object representing the operator and value by which the result is filtered with.
+ */
+data class FilterOperatorAndValue<T>(
+    val operator: SqlOperator,
+    val value: T
+)
+
+@Serializable
+/**
+ * An enumeration of SQL operators.
+ */
+enum class SqlOperator {
```

Comment:
I think I would prefer a more generic name like `ComparisonOperator`, at least in the API module. Mentioning SQL here exposes an implementation detail and they might in future also be used for requests that do get their data from the database, like the user endpoints.